debezium-server/debezium-server-sink-pubsub/config-mysql/application.properties

```diff
@@ -0,0 +1,12 @@
+debezium.sink.type=pubsub
```

Comment:
Did you try to run it? Does it really work without internal schema history configured?

Reply:
Edited. I also tested the connector with the result event was successfully sent to Pubsub.